test/testchecksumvalidator.cpp

```diff
@@ -180,7 +180,7 @@ using namespace OCC::Utility;
 #else
         auto *vali = new ValidateChecksumHeader(this);
         connect(vali, SIGNAL(validated(QByteArray,QByteArray)), this, SLOT(slotDownValidated()));
-        connect(vali, SIGNAL(validationFailed(QString)), this, SLOT(slotDownError(QString)));
+        connect(vali, SIGNAL(validationFailed(QString, QByteArray, QByteArray, QString)), this, SLOT(slotDownError(QString, QByteArray, QByteArray, QString)));
```

Comment:
why not use the syntax without the macros?

Reply:
@FlexW I thought I don't want to change it as then, I will have to change 2 other connects too.